build.gradle

```
@@ -39,6 +39,8 @@ dependencies {
     implementation 'org.json:json:20190722'
     implementation 'org.jetbrains:annotations:15.0'
     implementation 'com.google.code.gson:gson:2.8.6'
+
+    compile 'com.google.guava:guava:30.0-jre'
```

```suggestion
    implementation 'com.google.guava:guava:30.0-jre'
```

`compile` ist alte Gradle-Syntax